### DISCUSSION TITLE
[[START HERE]] -- Welcome to FAQ Discussions!

### DISCUSSION THREAD

**Discussion by tallguyjenks** [BryanJenksCommunity/FAQ#20] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by clyde67** [comment]
Hi, I wanted to know where do I put my notes? I've tried a folder system and it really didn't work out so I was wondering, if I use #tags to sort them should I have an inbox placeholder for new notes that haven't been processed and an index placeholder for everything putting everything rather than just having then listed in the vault?
